docs/LogGuide.md

```diff
@@ -2821,3 +2853,399 @@ Parsed Log Line Examples:
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>266)\|(?<timestamp>[^|]*)\|(?<npcId>[^|]*)\|(?<npcNameId>[^|]*)\|(?<npcYellId>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 266 (?<type>10A):(?<npcId>[^:]*):(?<npcNameId>[^:]*):(?<npcYellId>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+266|2024-02-29T15:15:40.5850000-08:00|4001F001|02D2|07AF|8f731e1760bdcfc9
+266|2024-02-29T15:15:54.5570000-08:00|4001F002|02D4|07BE|ae0674ec1e496642
+266|2024-02-25T16:02:15.0300000-05:00|E0000000|6B10|2B29|65aa9c0faa3d0e16
+
+Parsed Log Line Examples:
+[15:15:40.585] 266 10A:4001F001:02D2:07AF
+[15:15:54.557] 266 10A:4001F002:02D4:07BE
+[16:02:15.030] 266 10A:E0000000:6B10:2B29
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=NpcYell&lang=en-US) -->
+
+<a name="line267"></a>
+
+### Line 267 (0x10B): BattleTalk2
+
+This log line is emitted whenever a BattleTalk2 packet is received from the server,
+resulting in popup dialog being displayed during instanced content.
+
+`npcNameId` and `instanceContentTextId` (both hex values) correspond to IDs
+in the [BNpcName](https://github.com/xivapi/ffxiv-datamining/blob/master/csv/BNpcName.csv)
+and [InstanceContentTextData](https://github.com/xivapi/ffxiv-datamining/blob/master/csv/InstanceContentTextData.csv)
+tables, respectively.
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=BattleTalk2&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+267|[timestamp]|[npcId]|[instance]|[npcNameId]|[instanceContentTextId]|[displayMs]
+
+Parsed Log Line Structure:
+[timestamp] 267 10B:[npcId]:[instance]:[npcNameId]:[instanceContentTextId]:[displayMs]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>267)\|(?<timestamp>[^|]*)\|(?<npcId>[^|]*)\|(?<instance>[^|]*)\|(?<npcNameId>[^|]*)\|(?<instanceContentTextId>[^|]*)\|(?<displayMs>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 267 (?<type>10B):(?<npcId>[^:]*):(?<instance>[^:]*):(?<npcNameId>[^:]*):(?<instanceContentTextId>[^:]*):(?<displayMs>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+267|2024-02-29T16:22:41.4210000-08:00|00000000|80034E2B|02CE|840C|5000|0|2|0|0|6f6ccb784c36e978
+267|2024-02-29T16:22:17.9230000-08:00|00000000|80034E2B|02D2|8411|7000|0|2|0|0|be1dee98cdcd67a4
+267|2024-02-29T16:23:00.6680000-08:00|4001FFC4|80034E2B|02D5|840F|3000|0|2|0|0|cffef89907b5345b
+
+Parsed Log Line Examples:
+[16:22:41.421] 267 10B:00000000:80034E2B:02CE:840C:5000:0:2:0:0
+[16:22:17.923] 267 10B:00000000:80034E2B:02D2:8411:7000:0:2:0:0
+[16:23:00.668] 267 10B:4001FFC4:80034E2B:02D5:840F:3000:0:2:0:0
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=BattleTalk2&lang=en-US) -->
+
+<a name="line268"></a>
+
+### Line 268 (0x10C): Countdown
+
+This log line is emitted whenever a countdown is started.
+
+`result` is `00` if successful, and non-zero if the attempt to start a countdown failed
+(e.g., if a countdown is already in progress, or if combat has started).
```

Comment:
It may be worth it to note here that there is no network packet for the countdown finishing, and therefore no line will be (or can reasonably be) emitted for the `Engage!` log line.